composer/utils/checkpoint.py

```diff
@@ -1013,6 +1013,15 @@ def safe_torch_load(
                 'pass `load_ignore_keys = ["state/train_metrics/*", "state/eval_metrics/*"]`.',
             ) from e
         raise e
+    except FileNotFoundError as e:
+        if 'No such file or directory' in str(e) and dist.get_local_rank() != 0:
+            global_rank_zeor = dist.get_global_rank() - dist.get_local_rank()
+            raise FileNotFoundError(
+                f'No such file or directory: {e.filename}. '
+                f'This likely implies a download failed on local rank 0 or global rank {global_rank_zeor}'
+                f'Please check the logs for global rank {global_rank_zeor} to debug the checkpoint download issue.',
+            ) from e
+        raise e
```

Comment:
```suggestion
            global_rank_zero = dist.get_global_rank() - dist.get_local_rank()
            raise FileNotFoundError(
                f'No such file or directory: {e.filename}. '
                f'This likely implies a download failed on local rank 0 or global rank {global_rank_zero}'
                f'Please check the logs for global rank {global_rank_zero} to debug the checkpoint download issue.',
            ) from e
        raise e
```
error should point to local_rank_0, not global rank 0 since its different filesystem. but error log says look at global_rank_0 which is wrong